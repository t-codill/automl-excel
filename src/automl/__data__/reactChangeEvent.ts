// tslint:disable: no-any
export const reactChangeEvent: React.ChangeEvent<any> = {
    target: {}
} as unknown as React.ChangeEvent<any>;
