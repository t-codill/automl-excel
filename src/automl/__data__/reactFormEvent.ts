// tslint:disable: no-any
export const reactFormEvent: React.FormEvent<any> = {
    preventDefault(): void { return; }
} as unknown as React.FormEvent<any>;
