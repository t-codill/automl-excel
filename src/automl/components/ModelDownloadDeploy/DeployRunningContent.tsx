import { RunHistoryAPIsModels } from "@vienna/runhistory";
import { MessageBar, MessageBarType } from "office-ui-fabric-react";
import * as React from "react";

interface IDeployRunningContentProps {
    run: RunHistoryAPIsModels.MicrosoftMachineLearningRunHistoryContractsRunDto;
    deployName: string;
    startTime: string;
}
export class DeployRunningContent extends React.Component<IDeployRunningContentProps> {
    public render = (): React.ReactNode => {
        return (
            <>
                <MessageBar messageBarType={MessageBarType.info}>
                    Model of Iteration<strong> {this.props.run.properties && this.props.run.properties.iteration} </strong> is deploying to <strong>{this.props.deployName}</strong>.
                    This can take more than 20 minutes to complete.<br /> <br />
                    Deployment started on {this.props.startTime}.
                </MessageBar>
            </>
        );
    }
}
