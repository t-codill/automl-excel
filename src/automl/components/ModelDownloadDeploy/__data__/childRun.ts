import { RunHistoryAPIsModels } from "@vienna/runhistory";

export const childRun: RunHistoryAPIsModels.MicrosoftMachineLearningRunHistoryContractsRunDetailsDto = {
    runId: "test-run-id_1",
    properties: {
        DataPrepJsonString: "{\"features\":[\"timeStamp\",\"precip\",\"temp\"]}"
    }
};
