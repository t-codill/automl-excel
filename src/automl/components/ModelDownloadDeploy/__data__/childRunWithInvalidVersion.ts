import { RunHistoryAPIsModels } from "@vienna/runhistory";

export const childRunWithInvalidVersion: RunHistoryAPIsModels.MicrosoftMachineLearningRunHistoryContractsRunDetailsDto = {
    runId: "test-run-id_1",
    properties: {
        dependencies_versions: "{\"test\": \"1.0.0\"}"
    }
};
