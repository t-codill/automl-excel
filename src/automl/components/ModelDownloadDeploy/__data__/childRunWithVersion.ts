import { RunHistoryAPIsModels } from "@vienna/runhistory";

export const childRunWithVersion: RunHistoryAPIsModels.MicrosoftMachineLearningRunHistoryContractsRunDetailsDto = {
    runId: "test-run-id_1",
    properties: {
        dependencies_versions: "{\"azureml-train-automl\": \"1.0.0\"}"
    }
};
