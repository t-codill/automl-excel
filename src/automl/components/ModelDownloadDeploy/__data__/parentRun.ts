import { RunHistoryAPIsModels } from "@vienna/runhistory";

export const parentRun: RunHistoryAPIsModels.MicrosoftMachineLearningRunHistoryContractsRunDetailsDto = {
    runId: "test-run-id",
    properties: {
        DataPrepJsonString: "{\"features\":[\"timeStamp\",\"precip\",\"temp\"]}"
    }
};
