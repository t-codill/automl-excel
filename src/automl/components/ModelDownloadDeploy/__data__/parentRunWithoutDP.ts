import { RunHistoryAPIsModels } from "@vienna/runhistory";

export const parentRunWithoutDP: RunHistoryAPIsModels.MicrosoftMachineLearningRunHistoryContractsRunDetailsDto = {
    runId: "test-run-id",
    properties: {
    }
};
