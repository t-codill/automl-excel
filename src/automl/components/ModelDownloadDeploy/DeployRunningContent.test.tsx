import { shallow } from "enzyme";
import * as React from "react";
import { childRun } from "./__data__/childRun";
import { DeployRunningContent } from "./DeployRunningContent";

describe("Deploy Succeed Content", () => {
    it("should render with valid property", () => {
        const tree = shallow(<DeployRunningContent
            deployName="AAA"
            run={childRun}
            startTime="test-start-time"
        />);
        expect(tree)
            .toMatchSnapshot();
    });
});
