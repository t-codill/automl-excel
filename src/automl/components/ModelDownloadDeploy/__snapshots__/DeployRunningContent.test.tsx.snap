// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Deploy Succeed Content should render with valid property 1`] = `
<Fragment>
  <StyledMessageBarBase
    messageBarType={0}
  >
    Model of Iteration
    <strong>
       
       
    </strong>
     is deploying to 
    <strong>
      AAA
    </strong>
    . This can take more than 20 minutes to complete.
    <br />
     
    <br />
    Deployment started on 
    test-start-time
    .
  </StyledMessageBarBase>
</Fragment>
`;
